src/bin/nydusd/main.rs

```diff
@@ -210,7 +210,7 @@ fn append_fs_options(app: App<'static, 'static>) -> App<'static, 'static> {
         Arg::with_name("shared-dir")
             .long("shared-dir")
             .short("s")
-            .help("Directory shared by host and guest for passthroughfs, which also enables pathroughfs mode")
+            .help("Directory shared by host and guest for passthroughfs or a passthroughfs source for FUSE(for test and demonstration purpose), which also enables passthroughfs mode")
```

Comment:
should we a space and  (for the purpose of testing and demonstration)

Reply:
Sure, leaving a white space makes it neater.

Reply:
Then the help message looks like  below:
![image](https://user-images.githubusercontent.com/23294915/171126929-3335213d-b06e-45f1-96df-60a7c7ccb405.png)